pinot-core/src/main/java/org/apache/pinot/server/realtime/ServerSegmentCompletionProtocolHandler.java

```diff
@@ -52,6 +53,7 @@ public class ServerSegmentCompletionProtocolHandler {
   private static final String HTTP_PROTOCOL = CommonConstants.HTTP_PROTOCOL;
 
   private static SSLContext _sslContext;
+  private static HttpClientConfig _httpClientConfig = HttpClientConfig.DEFAULT_HTTP_CLIENT_CONFIG;
```

Comment:
I don't follow the changes in this file

Reply:
So right now most of the member vars are stored in a static context in this class, and Pinot server calls `ServerSegmentCompletionProtocolHandler#init` on start-up to make sure everything is assigned some values.

Without setting this to the default value, some of the UTs were failing because we were not initializing `ServerSegmentCompletionProtocolHandler` in them. So I had two options:

1. Either initialize it everywhere in the test
2. or go with a default value.

I picked option-2. lmk if you see any concerns.